recipe/meta.yaml

```diff
@@ -1,18 +1,17 @@
 {% set name = "bokeh" %}
-{% set version = "3.0.3" %}
+{% set version = "3.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bokeh-{{ version }}.tar.gz
-  sha256: 1c28471ef5e6110ba5bed513137fd26054ebc4454bc768650eaeefc53b898a8a
+  sha256: 9047dfe50a671b5e19ca588fd90a8e6bff197114a5dbd6a59cc2678942c27887
 
 build:
   number: 0
-  # Currently xyzservices >=2021.09.1 is not available on s390x
-  skip: true  # [py<38 or s390x]
+  skip: true  # [py<38]
```

Comment:
`Bokeh` has recently dropped support for python versions less than 3.9. As a result, it might be good to fix `line 14` to  `skip: true # [py<39]` to reflect this. :)  See documentation [here.](https://github.com/bokeh/bokeh/blob/cc911148cf5653feea7422aa0eee0cb5d221fd59/pyproject.toml#L10)

Reply:
It's from the main branch but the tag 3.1.0 still have support of python 3.10 https://github.com/bokeh/bokeh/blob/3.1.0/pyproject.toml

Reply:
yes, I also almost caught by that :)